src/Sarif.Viewer.VisualStudio.UnitTests/SdkUiUtilitiesTests.cs

```diff
@@ -128,6 +150,32 @@ public void SarifSnapshot_GetMessageEmbeddedLinkInlines_OneLinkPlusEscapedBracke
             VerifyTextRun(expected[2], actual[2]);
         }
 
+        [Fact]
+        public void SarifSnapshot_GetMessageEmbeddedLinkInlines_WebLink()
+        {
+            string url = "http://sarifweb.azurewebsites.net";
```

Comment:
>http://sarifweb.azurewebsites.net [](start = 26, length = 33)

i suggest using [https://example.com](https://example.com).

Secure link! intended for test/docs purposes. :)

Reply:
Right.